src/module/actor/character/apps/formula-picker/app.svelte

```diff
@@ -66,7 +66,11 @@
                         <span>{formula.item.name}</span>
                         <ItemTraits traits={formula.item.traits} rarity={formula.item.rarity} />
                     </div>
-                    <button onclick={() => confirmSelection(formula.item.uuid)} data-tooltip="Confirm">
+                    <button
+                        onclick={() => confirmSelection(formula.item.uuid)}
+                        aria-label={game.i18n.localize("Confirm")}
```

Comment:
I've been thinking we would have to cave, but would aria-describedby="tooltip" or aria-labelledby="tooltip" work instead? Foundry injects the former.

Reply:
labelledby works, describedby still produces the error.